app/src/main/java/com/github/se/polyfit/viewmodel/post/CreatePostViewModel.kt

```diff
@@ -26,6 +32,16 @@ constructor(
 
   suspend fun getRecentMeals() = mealRepository.getAllMeals().sortedBy { it.createdAt }.take(5)
 
+  private val _meals = MutableStateFlow<List<Meal>>(emptyList())
+  val meals = _meals
+
+  init {
+    viewModelScope.launch {
```

Comment:
Just need to be careful with using the `viewModelScope`. It caused a previous issue with the `CreateMealViewModel` not being synchronized correctly. But it should be okay as a new `ViewModel` is created each time by Hilt.

Reply:
Will do!